CMake/FindMiniupnpc.cmake

```diff
@@ -17,3 +17,11 @@ find_package_handle_standard_args(MINIUPNPC DEFAULT_MSG MINIUPNPC_INCLUDE_DIR MI
 set(MINIUPNPC_LIBRARIES ${MINIUPNPC_LIBRARY})
 set(MINIUPNPC_INCLUDE_DIRS ${MINIUPNPC_INCLUDE_DIR})
 mark_as_advanced(MINIUPNPC_INCLUDE_DIR MINIUPNPC_LIBRARY MINIUPNPC_API_VERSION_STR)
+
+if (MINIUPNPC_FOUND AND NOT TARGET miniupnpc)
+  add_library(miniupnpc UNKNOWN IMPORTED)
+  set_target_properties(miniupnpc PROPERTIES
```

Comment:
`add_library(Miniupnpc::miniupnpc UNKNOWN IMPORTED)`  in the Find module.
`add_library(Miniupnpc::miniupnpc ALIAS miniupnpc)` in its CMakeLists.txt